CHANGELOG.md

```diff
@@ -308,7 +308,7 @@ Please note that there are many excellent integrations for Prettier available, f
 - Add `--publish_dir_mode` back into the pipeline template ([nf-core/rnaseq#752](https://github.com/nf-core/rnaseq/issues/752#issuecomment-1039451607))
 - Add optional loading of of pipeline-specific institutional configs to `nextflow.config`
 - Make `--outdir` a mandatory parameter ([nf-core/tools#1415](https://github.com/nf-core/tools/issues/1415))
-- Add description between double quotes to avoid errors with apostrophes in a description ([#2066](https://github.com/nf-core/tools/pull/2066))
+- Add pipeline description and authors between double quotes to avoid errors with apostrophes ([#2066](https://github.com/nf-core/tools/pull/2066), [#2104](https://github.com/nf-core/tools/pull/2104))
```

Comment:
```suggestion
- Add pipeline description and authors between triple quotes to avoid errors with apostrophes ([#2066](https://github.com/nf-core/tools/pull/2066), [#2104](https://github.com/nf-core/tools/pull/2104))
```